crates/cxx-qt-gen/src/generator/structuring/mod.rs

```diff
@@ -54,6 +51,113 @@ impl<'a> Structures<'a> {
             }
         }
 
+        // Associate each method parsed with its appropriate qobject
+        for method in &cxxqtdata.methods {
+            if let Some(qobject) = qobjects
+                .iter_mut()
+                .find(|qobject| qobject.has_qobject_name(&method.qobject_ident))
+            {
+                qobject.methods.push(method);
+            } else {
+                return Err(Error::new_spanned(
+                    &method.qobject_ident,
+                    format!("Unknown QObject: {:?}", &method.qobject_ident),
+                ));
```

Comment:
Wonder if this is more readable then these big if let Some blocks ?

```rust
            let mut qobject = qobjects
                .iter_mut()
                .find(|qobject| qobject.has_qobject_name(qobject_ident))
                .ok_or_else(|| {
                    Error::new_spanned(qobject_ident, format!("Unknown QObject: {qobject_ident}"))
                })?;
            qobject.methods.push(method);
```

Reply:
Done